src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/FilterDelegationBase.cs

```diff
@@ -187,6 +183,13 @@ protected bool IsValidDelegatableFilterPredicateNode(TexlNode dsNode, TexlBindin
 
             return true;
         }
+
+        private bool IsNodeBooleanOptionSetorBooleanFieldorView(TexlNode dsNode, TexlBinding binding)
```

Comment:
>IsNodeBooleanOptionSetorBooleanFieldorView [](http://example.com/codeflow?start=21&length=42)

This name is a little clunky.  Would it be better to simplify it to something like IsNodeDelegableBooleanOrView